web/src/popover_menus.ts

```diff
@@ -332,10 +332,14 @@ export function on_show_prep(instance: tippy.Instance): void {
 function get_props_for_popover_centering(
     popover_props: Partial<tippy.Props>,
 ): Partial<tippy.Props> {
+    let placement: tippy.Placement = "top";
+    if (util.is_mobile()) {
+        placement = "bottom";
+    }
```

Comment:
I am not sure what's happening but I can verify the fix. Since we are resetting the reference to (0, 0) in DOM and centering the popover. The `placement` here doesn't matter. So, using either `top` or `bottom` is fine. So, let's just change to use `bottom` here with a comment that this works well with popper's positioning system when the popover exceeds window height.